webpack/JobWizard/steps/AdvancedFields/__tests__/AdvancedFields.test.js

```diff
@@ -297,4 +307,30 @@ describe('AdvancedFields', () => {
       }).value
     ).toBe('Run Default val');
   });
+
+  it('search resources action', async () => {
+    store.clearActions();
+    jest.useFakeTimers();
+
+    render(
+      <Provider store={store}>
+        <JobWizard />
+      </Provider>
+    );
+    await act(async () => {
+      fireEvent.click(screen.getByText(WIZARD_TITLES.advanced));
```

Comment:
I see it in `Target hosts and inputs` tab, should it also appear in `Advanced Fields`?

Reply:
It comes from the same function in both: `formatter`